cmd/secretless-broker/main.go

```diff
@@ -2,12 +2,48 @@ package main
 
 import (
 	"flag"
+	"os"
+	"os/signal"
+	"syscall"
 
-	"github.com/cyberark/secretless-broker/internal/plugin"
 	"github.com/cyberark/secretless-broker/pkg/secretless"
+	v2 "github.com/cyberark/secretless-broker/pkg/secretless/config/v2"
+	"github.com/pkg/profile"
 )
 
 func main() {
```

Comment:
This PR undoes partially the move of main broker code from cmd/ to pkg/. We should ensure that only flag parsing is done in main.go so that if needed, the broker can be run/started from a package that imports it.

Reply:
I see what you mean.  I can go ahead and move all this stuff into `Secretless` per my original plan.